content/_index.html

```diff
@@ -9,117 +9,102 @@
 ---
 
 <h2 class="mbs">Explore our APIs</h2>
-<div class="flex flex-wrap mbxl">
-  <a
-    href='{{< relref "api/shipping-guide_2" >}}'
-    class="mb-card flex flex-dir-col flex-1-1-20r pam mrxs mbxs"
-  >
+<div class="flex flex-wrap gaxs mbxl">
+  <div class="bg-green1 flex flex-dir-col start__col pam">
     <div class="flex align-ic mlxs">
       <span
         data-mybicon="mybicon-list"
         data-mybicon-class="icon-ui--green-dark mrs"
         data-mybicon-width="19"
         data-mybicon-height="19"
       ></span>
-      <h3 class="mlxs fwn">Shipping Guide API</h3>
+      <h3 class="mlxs fwn">Checkout</h3>
     </div>
     <p class="mhxs pll">
       Look up available services, delivery times, prices and environmental data,
       and use our recommended service descriptions in your checkout.
```

Comment:
Something like this, and then add the API links below, similar to what’s in the testing page. I think it gets too messy if we try to incorporate each API into a sentence without sounding like a sales pitch from outer space.

```suggestion
      Look up available services, delivery times, prices and environmental data. Verify addresses and let users pick the closest pickup point. And use our recommended service descriptions in your checkout.
```